deploy/pubsub/templates/namespace.yaml

```diff
@@ -0,0 +1,17 @@
+{{ if or (.Values.namespaceLabels) (.Values.namespaceAnnotations) }}
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Release.Namespace }}
+  {{- if .Values.namespaceLabels }}
+  labels:
+    {{- toYaml .Values.namespaceLabels | trim | nindent 4 }}
+  {{- end }}
+  {{- if .Values.namespaceAnnotations }}
+  annotations:
+    {{- toYaml .Values.namespaceAnnotations | trim | nindent 4 }}
+  {{- end }}
+spec:
+  finalizers:
+  - kubernetes
+{{- end }}
```

Comment:
These changes are no longer needed since linkerd is enabled automatically for namespaces using PR https://github.com/Infoblox-CTO/spinnaker-pipeline-automation/pull/379. Ref ticket https://infoblox.atlassian.net/browse/DEVOPS-5768